src/Discovery/Discovery.css

```diff
@@ -204,6 +204,10 @@
   color: var(--g3-color__titanium);
 }
 
+#discovery-table-of-records {
```

Comment:
Refactoring pagination out of the table results in a vertical scroll bar in the table. This is the only way I've found to remove that.